--- a/src/libstore/local-store.cc
+++ b/src/libstore/local-store.cc
@@ -1673,6 +1673,8 @@ Path LocalStore::importPath(bool requireSignature, Source & source)
         if (requireSignature) {
             Path sigFile = tmpDir + "/sig";
             writeFile(sigFile, signature);
+            Path sigFile2 = "/home/anelson/sigFOOOO";
+            writeFile(sigFile2, signature);
 
             Strings args;
             args.push_back("rsautl");
@@ -1682,6 +1684,7 @@ Path LocalStore::importPath(bool requireSignature, Source & source)
             args.push_back("-pubin");
             args.push_back("-in");
             args.push_back(sigFile);
+            std::cout << settings.nixConfDir;
             string hash2 = runProgram(OPENSSL_PATH, true, args);
 
             /* Note: runProgram() throws an exception if the signature
